src/common/font.c

```diff
@@ -56,33 +60,30 @@ font_width(struct font *font, const char *string)
 	return rectangle.width;
 }
 
-void
-font_buffer_update(struct lab_data_buffer **buffer, int max_width,
-	const char *text, struct font *font, float *color, double scale)
-{
-	if (*buffer) {
-		wlr_buffer_drop(&(*buffer)->base);
-		*buffer = NULL;
-	}
-	font_buffer_create(buffer, max_width, text, font, color, scale);
-}
-
 void
 font_buffer_create(struct lab_data_buffer **buffer, int max_width,
-	const char *text, struct font *font, float *color, double scale)
+	const char *text, struct font *font, float *color, const char *arrow,
+	double scale)
 {
+	assert(max_width);
```

Comment:
Does this still work with setting any of these theme `size` vars to `0`?
- `ActiveWindow`
- `MenuItem`
- `OSD`